samples/posix/eventfd/README.rst

```diff
@@ -0,0 +1,46 @@
+.. _posix-eventfd-sample:
+
+POSIX eventfd()
+########################################################
+
+Overview
+********
+
+This sample application demonstrates using the POSIX eventfd()
+function to create a file descriptor, which can be used for event notification.
+The returned file descriptor is used with write/read calls and write/read values
+are output to the console.
+
+
+Building and Running
+********************
+
+This project outputs to the console.  It can be built and executed
+on QEMU as follows:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/posix/eventfd
+   :host-os: unix
+   :board: qemu_x86
+   :goals: run
+   :compact:
+
+For comparison, to build directly for your host OS if it is POSIX compliant (for ex. Linux):
+
+.. code-block:: console
+
+   make -f Makefile.host
```

Comment:
Thanks for pointing bringing this to my attention. It looks like the Makefile.host file is actually set up to work only if the cwd is **samples/posix/eventfd**, so the command to build it would be `make -f Makefile.host`. I can update the Makefile.host files so it can be built with  `make -f samples/posix/eventfd/Makefile.host`, similar to how the west build would work, and update the instructions.